Source/AwsCommonRuntimeKit/event-stream/EventStreamMessage.swift

```diff
@@ -54,36 +54,37 @@ extension EventStreamMessage {
         }
         let addCHeader: () throws -> Int32 = {
             return try header.name.withCString { headerName in
+                let headerNameLength = UInt8(header.name.utf8.count)
                 switch header.value {
                 case .bool(let value):
                     return aws_event_stream_add_bool_header(
                         rawHeaders,
                         headerName,
-                        UInt8(header.name.count),
+                        headerNameLength,
```

Comment:
This is spooky. Please search the codebase for all other uses of `withCString` or `.count` to see if we're making this mistake anywhere else

Reply:
Thanks, I have removed `withCString` since it is not needed and Swift will automatically do it for you under the hood. I have also verified the usage of `.count`.